ambry-tools/src/main/java/com/github/ambry/clustermap/HelixVcrPopulateTool.java

```diff
@@ -145,7 +146,7 @@ public static void main(String[] args) {
   static void createCluster(String destZkString, String destClusterName) {
     HelixZkClient destZkClient = getHelixZkClient(destZkString);
     HelixAdmin destAdmin = new ZKHelixAdmin(destZkClient);
-    if (destAdmin.getClusters().contains(destClusterName)) {
+    if (ZKUtil.isClusterSetup(destClusterName, destZkClient)) {
```

Comment:
@ankagrawal this minor change should help us get rid of the warning messages we saw when building a new VCR cluster.